src/mobx/stores/PricesStore.ts

```diff
@@ -20,10 +20,12 @@ export default class PricesStore {
   loadPrices = action(async (): Promise<void> => {
     const prices = await this.store.api.loadPrices();
     if (prices) {
-      this.priceCache = {
-        ...this.priceCache,
-        ...prices,
-      };
+      runInAction(() => {
```

Comment:
Same as above 

This statement is already in action. Do we still need it to be in runInAction